app/src/Pages/Recipes/Components/CreateRecipe.js

```diff
@@ -132,142 +63,7 @@ export default () => {
-            id="prepTime"
-            autoComplete="prepTime"
-            onChange={e => setPrepTime(e.target.value)}
-          />
-          <TextField
-            variant="outlined"
-            margin="normal"
-            fullWidth
-            name="servings"
-            label="Servings"
-            id="servings"
-            autoComplete="servings"
-            onChange={e => setServings(e.target.value)}
-          />
-
-          {ingredients.map((ingredient,index) => (
-            <Fragment key={`${index}`}>
-                <TextField
-                    variant="outlined"
-                    margin="normal"
-                    fullWidth
-                    name="ingredient"
-                    label="Ingredient"
-                    id="ingredient"
-                    onChange={event => handleIngredientChange(index, event)}
-                />
-              <div className="form-group col-sm-2">
-                <button
-                  className="btn btn-link"
-                  type="button"
-                  onClick={() => handleRemoveIngredients(index)}
-                >
-                  -
-                </button>
-                <button
-                  className="btn btn-link"
-                  type="button"
-                  onClick={() => handleAddIngredients()}
-                >
-                  +
-                </button>
-              </div>
-            </Fragment>
-          ))}
-         
-         {directions.map((direction,index) => (
-            <Fragment key={`${index}`}>
-               <TextField
-                    variant="outlined"
-                    margin="normal"
-                    fullWidth
-                    name="direction"
-                    label="Direction"
-                    id="direction"
-                    onChange={event => handleDirectionChange(index, event)}
-                />
-              <div className="form-group col-sm-2">
-                <button
-                  className="btn btn-link"
-                  type="button"
-                  onClick={() => handleRemoveDirections(index)}
-                >
-                  -
-                </button>
-                <button
-                  className="btn btn-link"
-                  type="button"
-                  onClick={() => handleAddDirections()}
-                >
-                  +
-                </button>
-              </div>
-            </Fragment>
-          ))}
-
-          <Button
-                type="submit"
-                fullWidth
-                variant="contained"
-                color="primary"
-                className={formClasses.submit}
-                > 
-                    Save Recipe
-         </Button>
-         <Button
-                type="button"
-                fullWidth
-                className={formClasses.submit}
-                variant="contained"
-                color="secondary"
-                href="./"
```

Comment:
I don't know if using href is the appropriate choice for routing in this use case. It appears when the cancel button is clicked it is making an HTTP request. I noticed the username was getting lost, I think this is due to the page reloading. On the login components, I handled this by using an onclick() and then navigating with react router's "redirect tag"